### DISCUSSION TITLE
How to solve "You have been blocked. Why? Something about the behaviour of the browser has caught our attention...

### DISCUSSION THREAD

**Discussion by truongthicam** [cypress-io/cypress#29363] [Questions and Help]
Please help me solve it
While testing sign in feature the browser said:

You have been blocked.
Why? Something about the behaviour of the browser has caught our attention.

There are various possible explanations for this:

 You are browsing and clicking at a speed much faster than expected of a human being.
 Something is preventing JavaScript from working on your computer.
 There is a robot on the same network (IP x.x.x.x) as you.
Having problems accessing the site? Submit feedback.

ID: ...

**Comment by portyu9** [comment]
Hi @truongthicam 

That message is coming from the website/security layer protecting the site, not from a Cypress assertion.

The site is detecting the automated browser, IP/network traffic, or some other characteristic and is intentionally blocking the request.

If this is an application you control, the recommended solution is to configure your test environment/security provider to allow Cypress test traffic (for example, a test-environment bypass or allowlist) rather than trying to defeat the bot protection from Cypress.

If this is a third-party authentication/site, use its supported test/programmatic authentication mechanism where possible instead of automating through its anti-bot challenge.

Options such as:

```js
chromeWebSecurity: false
```

do not solve server-side bot/WAF classification.

The block page's feedback/request ID may also be useful to whoever manages the site's security service to determine exactly which rule rejected the test.

Please mark this answer as accepted if it helped, thank you.
